src/main/java/frc/robot/subsystems/AlignerSubsystem.java

```diff
@@ -0,0 +1,103 @@
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.AlignerConstants.*;
+
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.motorcontrol.MotorUtil;
+import frc.robot.shuffleboard.GRTNetworkTableEntry;
+import frc.robot.shuffleboard.GRTShuffleboardTab;
+
+public class AlignerSubsystem extends SubsystemBase {
+    // current target position for motors
+    public double current_slaptarget = OPENSLAP;
+    public double current_anglertarget = OPENANGLER;
+    // current action state
+    public boolean to_slap;
+    public boolean to_grab;
+    // state of angler(open or closed)
+    public boolean angler_open = true;
+
+    private final WPI_TalonSRX motorSlapper = MotorUtil.createTalonSRX(SLAPID);
+    private final WPI_TalonSRX motorAngler = MotorUtil.createTalonSRX(ANGLEID);
+
+    private final GRTShuffleboardTab shuffleboardTab = new GRTShuffleboardTab("Aligner");
+    private final GRTNetworkTableEntry slapperPositionEntry = shuffleboardTab.addEntry("Slapperposition", 0).at(0, 0);
+    private final GRTNetworkTableEntry anglerPositionEntry = shuffleboardTab.addEntry("Anglerposition", 0).at(1, 0);
+
+    // variables for motor positions
+    public double slappos;
+    public double anglerpos;
+
+    public AlignerSubsystem() {
+        // set motors to brake
+        motorSlapper.setNeutralMode(NeutralMode.Brake);
+        motorAngler.setNeutralMode(NeutralMode.Brake);
+
+        // setup encoders for motors
+        motorSlapper.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder);
+        motorSlapper.setSensorPhase(true);
+
+        motorAngler.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder);
+        motorAngler.setSensorPhase(true);
```

Comment:
usually best practice to also reset the encoder positions to 0 (just in case, i guess)
relative encoders (which we're using) are only capable of measuring rotations (well, 'ticks') since boot, so we'll need to make sure that the aligners start in a known 0 position and all our measurements are relative to that position